pybossa/api/__init__.py

```diff
@@ -397,3 +398,39 @@ def fetch_lock(task_id):
                       'expires': seconds_to_expire})
 
     return Response(res, 200, mimetype='application/json')
+
+
+@jsonpify
+@csrf.exempt
+@blueprint.route('/project/<int:project_id>/taskgold', methods=['POST'])
+@ratelimit(limit=ratelimits.get('LIMIT'), per=ratelimits.get('PER'))
+def task_gold(project_id=None):
+    """Make task gold"""
+    if not current_user.is_authenticated():
+        return abort(401)
+
+    project = project_repo.get(project_id)
+    if project is None or not(current_user.admin
+        or current_user.id in project.owners_ids):
```

Comment:
We could raise a `Forbidden` for the second condition

Reply:
done